src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+@import '../utils/variables';
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
+  gap: $calendar-gap;
+  width: $calendar-width;
+  transition: all 0.5s;
+
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $day-width;
+    height: $day-height;
+    background-color: #eee;
+    border: 1px solid $main-color;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+      }
+    }
+
+    color: $main-color;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-1px);
+    }
```

Comment:
due to task it should change a color and go up 20px when hover, please change it